src/OrchardCore.Cms.Web/Properties/launchSettings.json

```diff
@@ -12,7 +12,7 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Production"
```

Comment:
Why?

Reply:
Only testing code, need to be reverted

It prevents razor runtime compilation while in dev mode under VStudio